about/milestones.md

```diff
@@ -28,6 +28,7 @@ OpenMLDB 起源于业界领先的人工智能平台和服务提供商[第四范
 - 2021.12：举办国内首次 Feature Store Meetup
 - 2022.1：v0.4.0 发布，增强 SQL 为核心的开发体验，引入在线监控模块
 - 2022.1：OpenMLDB 第一次社区会议
+- 2022.2：v0.4.1 发布，主要为解决社区用户反馈的一些问题
```

Comment:
no need to put this minor version into the milestones, you can remove it :-)

Reply:
ok